src/robot.cpp

```
@@ -632,23 +632,22 @@ void Robot::executeManipulationPath()
   }
 }
 
-geometry_msgs::Pose Robot::getManipulationTarget()
+geometry_msgs::PoseStamped Robot::getManipulationTarget(const std::string& planning_group_name)
 {
-  std::string planning_group_name = config_->getFeatureManipulation().getActivePlanningGroup();
-  
-  auto group_it = planning_groups_.find(planning_group_name);
-  TEMOTO_INFO_STREAM(planning_group_name.c_str());
+  std::string planning_group = (planning_group_name.empty()) ? config_->getFeatureManipulation().getActivePlanningGroup() : planning_group_name;
```

Throw an error instead of using a default group.